static/app/views/performance/compare/content.tsx

```diff
@@ -58,7 +58,11 @@ class TransactionComparisonContent extends Component<Props> {
             <Breadcrumb
               organization={organization}
               location={location}
-              transactionName={transactionName}
+              // TODO: add this back in if transaction comparison is used
+              // transaction={{
+              //   project: <insert project id>,
+              //   name: transactionName,
+              // }}
```

Comment:
Disabling this in the meantime because fixing it would require more changes to some unused features.